packages/js/src/generators/init/schema.json

```
@@ -25,6 +25,11 @@
       "type": "string",
       "description": "Customize the generated tsconfig file name.",
       "x-priority": "internal"
+    },
+    "publishable": {
+      "type": "boolean",
+      "default": false,
+      "description": "Generate a publishable library."
     }
```

Publishable doesn't really make sense on `init` to me. We are creating a publishable library with the library generator, not the init generator.

At minimum, we could change the description to something like `Generate a publish config.`, but I think I'd go further and do something like this:

```suggestion
    "publishConfig": {
      "type": "boolean",
      "default": false,
      "description": "Generate a publish config."
    }
```

---

changed.